docs/kyma/docs/005-overview-kyma.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Kyma and Knative - brothers in arms
```

Comment:
please extend doc with https://github.com/kyma-project/website/pull/49/files just one paragraph, that nearest plans are not only Istion exchange but also replacement of Kubeless

Reply:
Will do.